xuemingjun/server/src/server_node.cpp

```diff
@@ -0,0 +1,39 @@
+//
+// Created by mijiao on 23-1-28.
+//
+#include <string>
+#include <map>
+#include <ros/ros.h>
+#include <client/message.h>
+#include <server/service.h>
+
+
+std::map<std::string, ros::Subscriber> sublist;
+ros::NodeHandle *nhp = nullptr;
+
+void topicCallBack(const client::message::ConstPtr &msg) {
+    ROS_INFO("From %s received: %d.%d", msg->name.c_str(), msg->time.sec, msg->time.nsec);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Lines should be <= 80 characters long  [whitespace/line_length] [2]